External/smoke/CMakeLists.txt

```diff
@@ -0,0 +1,572 @@
+# Smoke
+# https://github.com/ROCm-Developer-Tools/aomp.git
+
+include(External)
+
+option(TEST_SUITE_SMOKE_FORCE_ALL "Execute all Smoke tests, even those known to be unsupported by Clang" OFF)
+
+set(TEST_SUITE_OFFLOADING_FLAGS --offload-arch=native CACHE STRING "Compiler arguments for OpenMP offloading")
+set(TEST_SUITE_OFFLOADING_LDFLAGS --offload-arch=native CACHE STRING "Linker arguments for OpenMP offloading")
+
+set(INTEL_TEST_SUITE_EXPECT_PASS)
+
+set(AMD_TEST_SUITE_EXPECT_PASS
+    alignedattribute
+    already_locked
+    ancestorC
+    aomp-issue376
+    assertok_error
+    assume_no_openmp
+    atomic
+    atomic_double
+    big-jump-loop-1
+    big-jump-loop-collapse
+    big-jump-loop-env-vars
+    big-jump-loop-matmul
+    big-jump-loop-split-2
+    big-jump-loop-split-3
+    big-jump-loop-split-4
```

Comment:
These depend on downstream functionality and won't work with upstream. At least the filecheck in those test cases.

Reply:
updated with: cb9b0dbd31eea10325286bf389d0cac20cf33260